modules/oe_content_event/src/Plugin/Validation/Constraint/EventFieldsRequiredValidator.php

```diff
@@ -91,21 +91,37 @@ protected function validateGroupFieldsEmpty(array $fields, Constraint $constrain
    *   The node object.
    */
   protected function validateOrganiserGroupFields(Constraint $constraint, NodeInterface $node) {
-    $violation = NULL;
     if ($node->get('oe_event_organiser_internal')->isEmpty() && $node->get('oe_event_organiser_name')->isEmpty()) {
-      $violation = $this->context->buildViolation('You have to fill in at least one of the following fields: @internal or @organiser_name', [
+      return;
```

Comment:
Why do you return without anything? You could simply remove the whole if statement.

Reply:
Yes, it can be removed. Idea was to exit not to check next 3 conditions.